WerWolv98/EdiZon_ConfigsAndScripts
# Added Hyrule Warriors config

# ChangeLog
- With EdiZon 1.3.2, Hyrule Warriors editing no longer locks out the save file.  The original HW config was used as a base, with additional items added under "Character Edit".
- The README.md was updated to include the HW information.

# Previous Behavior
Prior to EdiZon 1.3.2, when playing Hyrule Warriors with an edited save, the save would not be updated with changes or progress, including settings.

# Expected Behavior
Using EdiZon 1.3.2 +, the software should be able to save changes or progress.

# Test Validation
1. Backup the save file.
1. Modify the save using EdiZon and the new .json config.
1. Load Hyrule Warriors, go to Settings, and change a setting.
1. Exit to main Switch Menu and close the Hyrule Warriors software.
1. Load Hyrule Warriors again, and verify that the changed setting persisted with the new value.
